pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -183,9 +183,13 @@ def add_edges_from(self, ebunch, **kwargs):
         for edge in ebunch:
             self.add_edge(edge[0], edge[1])
 
-    def get_intra_edges(self):
+    def get_intra_edges(self, time_slice=0):
         """
         returns the intra slice edges present in the 2-TBN.
+        Parameter
+        ---------
+        time_slice: The timeslice should be a positive value greater than or equal to zero
```

Comment:
@palashahuja Please check the way docstrings have been written. First line should be the data type and second line should be the description of the argument.